tests/test_api.py

```diff
@@ -180,8 +180,8 @@ def test_api_handler_send_api_request(api_handler, capsys):
     # of the _extra_fields attribute
 
     api_handler.endpoints = {
-        "catalog": "/intake/update",
-        "payu": "/payu/update",
+        "catalog": "intake/update",
+        "payu": "ayu/update",
```

Comment:
```suggestion
        "payu": "payu/update",
```
Not sure it matters here though what the end point is named :)

Reply:
Good spot - I'm terrible at typing